nvflare/fuel/hci/client/cli.py

```diff
@@ -133,6 +140,15 @@ def __init__(
             session_status_check_interval=1800,  # check server for session status every 30 minutes
             event_handlers=event_handlers,
         )
+
+        nvflare_dir = os.path.join(os.path.expanduser("~"), ".nvflare")
```

Comment:
probably it's better to place this history file in the admin tool workspace folder, instead of the account "~/.nvflare" folder.